src/main/java/com/gluonhq/substrate/Constants.java

```diff
@@ -90,12 +91,12 @@
 //    public static final String APP_LINUX = "linux";
 //    public static final String APP_IOS = "ios";
 //
-//    /**
-//     * Supported target source folders
-//     *
-//     */
-//    public static final String SOURCE_MAC = "mac";
-//    public static final String SOURCE_IOS = "ios";
+    /**
+     * Supported target source folders
+     *
+     */
+    public static final String SOURCE_MAC = "mac";
```

Comment:
why do you need something new here? Why not using OS?

Reply:
ok, replaced with triplet.getOs()`